src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.html

```diff
@@ -22,12 +22,16 @@
     <div class="dialog-content">
 
       <form [formGroup]="form">
-        <alg-collapsible-section i18n-title title="Can view" icon="fa fa-eye">
+        <alg-collapsible-section
+          i18n-title title="Can view" icon="fa fa-eye"
+          [collapsible]="!form.errors?.canView"
+        >
           <ng-template #content let-collapsed>
             <alg-progress-select
               [collapsed]="collapsed"
               formControlName="canView"
-              [values]="canViewValues"
+              [values]="progressSelectValues.canViewValues"
+              [validationErrors]="form.errors?.canView"
```

Comment:
In the PR #692, @ThomasJuster commented ([link of the thread here](https://github.com/France-ioi/AlgoreaFrontend/pull/692#discussion_r737474645))
> I don't understand the link with validation, to me validation is defined at form-level and errors are stored into `form.errors` object, into properties that may match input names
> 
> Here, input name and error property seem to match, so according to the [doc](https://angular.io/api/forms/AbstractControl#errors) we should be able to also access the error from the `canView` FormControl via property `errors`

I could access the form control form inside the component, but after some testing, the form group errors are not mapped to the form controls with the same name, so the only way to access the error, is by getting it from the parent.
Also I couldn't manage to display the errors after a change in the form, as the fields have no way of knowing the form has changed.